java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/OpenmldbBatchConfig.scala

```diff
@@ -120,7 +120,7 @@ class OpenmldbBatchConfig extends Serializable {
   @ConfigOption(name="openmldb.disable", doc="Disable OpenMLDB optimization or not")
   var disableOpenmldb = false
 
-  // HybridSE dynamic library path
+  // OpenMLDB Java SDK dynamic library path, notice that this should not be set hybridse jsdk so
```

Comment:
It should be noted that, we set `hybridseJsdkLibraryPath` to openmldb jsdk lib, because we can't load openmldb jsdk and hybridse jsdk at the same time. And openmldb jsdk has all symbols of hybridse jsdk, so it works.

Reply:
We will remove this config later since it has to be the same so file of openmldb-jdbc.